components/layout_2020/table/layout.rs

```diff
@@ -370,52 +370,50 @@ impl<'a> TableLayout<'a> {
         self.rows = vec![RowLayout::default(); self.table.size.height];
         self.columns = vec![ColumnLayout::default(); self.table.size.width];
 
+        let is_length = |size: &LengthPercentageOrAuto| {
+            size.non_auto().and_then(|size| size.to_length()).is_some()
```

Comment:
I'm not sure if something like the following is clearer or not:

```suggestion
            size.non_auto().is_some_and(|size| !size.has_percentage())
```

Reply:
Done, I forgot about `has_percentage()`